From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ikey Doherty <ikey@solus-project.com>
Date: Fri, 9 Dec 2016 02:38:46 +0000
Subject: [PATCH] Solus: Default font settings

Changes:
- Set default monospace font to hack
- Disable bitmap fonts besides emoji ones
---
 conf.d/60-latin.conf | 1 +
 conf.d/meson.build   | 1 +
 2 files changed, 2 insertions(+)

diff --git a/conf.d/60-latin.conf b/conf.d/60-latin.conf
index ff933af..fd8b8cc 100644
--- a/conf.d/60-latin.conf
+++ b/conf.d/60-latin.conf
@@ -35,6 +35,7 @@
 	<alias>
 		<family>monospace</family>
 		<prefer>
+			<family>Hack</family>
 			<family>Noto Sans Mono</family>
 			<family>DejaVu Sans Mono</family>
 			<family>Inconsolata</family>
diff --git a/conf.d/meson.build b/conf.d/meson.build
index b6d07ee..771fee9 100644
--- a/conf.d/meson.build
+++ b/conf.d/meson.build
@@ -66,6 +66,7 @@ conf_links = [
   '65-fonts-persian.conf',
   '65-nonlatin.conf',
   '69-unifont.conf',
+  '70-no-bitmaps-except-emoji.conf',
   '80-delicious.conf',
   '90-synthetic.conf',
 ]
